R/stats-anova-tidiers.R

```diff
@@ -127,8 +133,8 @@ tidy.anova <- function(x, ...) {
       ret < cbind(cbind(term, ret), response)
       row.names(ret) <- NULL
     }
-  } else if (length(grep("Model", x_attr$heading)) != 0) {
-    mods <- sub(".*: ", "", strsplit(x_attr$heading[grep("Model", x_attr$heading)], "\n")[[1]])
+  } else if (length(mod_lines) != 0) {
```

Comment:
```suggestion
  } else if (is.null(ret$term) & length(mod_lines) != 0) {
```

Reply:
This would be sufficient to ensure that merMod anova output isn't caught by any of these conditionals, I think.

Reply:
Unfortunately, I'm a fixed timeline to submit to CRAN due to [upcoming breaking changes from a dependency](https://github.com/tidymodels/broom/issues/1113). I will go ahead and move forward with this suggestion unless you'd like to move forward otherwise!

Reply:
If you've got a fix, that's great - go ahead!